inst/scripts/make-TreeSE-object.R

```diff
@@ -0,0 +1,45 @@
+
+library(tidyverse)
+library(remotes)
+library(mia)
+library(BiocManager)
+library(SummarizedExperiment)
+library(biomformat)
+library(TreeSummarizedExperiment)
+
+
+#Sample phenotype data
+samples <- read.csv("git_metadata_and_community_phenotypes.csv")
+
+#Abundance table
+counts <- readRDS("git_ASV_table.RDS")
+
+#Taxonomic mapping table
+tax <- counts %>% select(3:8)

```

Comment:
Select based on field names, not numbers. This is more explicit and less error-prone.